src/BlazorUI/Bit.BlazorUI/Components/Inputs/TextField/BitTextField.scss

```
@@ -334,24 +439,85 @@
     }
 
     .bit-tfl-lbl {
-        height: spacing(4);
-        margin: 0 spacing(1) 0 0;
-        line-height: spacing(2.75);
-        padding: spacing(0.625) 0 spacing(0.625) spacing(1.5);
+        height: var(--bit-tfl-min-height);
+        margin-inline-end: spacing(1);
+        // The label fills the height of the field minus its own padding, so its text stays centered
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)